website/content/docs/pipelines/config.mdx

```diff
@@ -0,0 +1,177 @@
+---
+layout: docs
+page_title: Pipelines Configuration
+description: |-
+  The page for detailing Waypoint pipeline configurations
+---
+
+# Pipelines Configuration
+
+This section details some of the various step built-in plugins and their
+configuration options.
```

Comment:
I cannot parse this sentence 😅

Reply:
I think I flipped some words.

> This section details some of the various *built-in step plugins* and their configuration options

Maybe?

Reply:
👍🏻